addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json

```diff
@@ -328,15 +340,25 @@
         1,
         6
       ],
+      "cookies": {
```

Comment:
This does not seem to be handled.

Reply:
It doesn’t seem supported by the upstream schema either (array or non-blank non-empty string).

Reply:
Edit: I'm wrong it is an object upstream.

Reply:
I'll tackle it. If I don't get it done shortly I'll open a new issue.